pkgs/servers/http/apache-modules/mod_cspnonce/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, apacheHttpd }:
+
+
+stdenv.mkDerivation rec {
+  pname = "mod_cspnonce";
+  version = "1.3";
+  src = fetchFromGitHub {
+    owner = "wyattoday";
+    repo = "mod_cspnonce";
+    rev = version;
+    sha256 = "0kqvxf1dn8r0ywrfiwsxryjrxii2sq11wisbjnm7770sjwckwqh5";
+  };
+
+  buildInputs = [ apacheHttpd ];
+
+  buildPhase = ''
+    export APACHE_LIBEXECDIR=$out/modules
+    export makeFlagsArray=(APACHE_LIBEXECDIR=$out/modules)
```

Comment:
Where is this used? `makeFlagsArray` would be passed to `make` in the default `buildPhase` and `installPhase` but here both are overridden.

Also missing hooks.

Reply:
I removed those. thanks for the review.